src/modules/blocks/rsvp/container.js

```diff
@@ -54,7 +54,7 @@ const setInitialState = ( dispatch, ownProps ) => () => {
 	dispatch( thunks.getRSVP( postId ) );
```

Comment:
If thunks was renamed to actions, should this be changed too?

Reply:
thunks is still used above in the import, so maybe it's just the method that got moved